backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java

```diff
@@ -95,6 +97,18 @@ public List<BucketResponse> listAllDeletedBuckets() {
         .collect(Collectors.toList());
   }
 
+  @Override
+  public void activateLambda(UUID bucketId, LambdaType lambdaType) {
+    BucketEntity bucketEntity = findBucketEntityById(bucketId);
+    List<LambdaType> activatedLambdas = bucketEntity.getLambdaTypes();
+    if (activatedLambdas.contains(lambdaType)) {
+      throw new LambdaAlreadyActivated();
+    }
+    activatedLambdas.add(lambdaType);
```

Comment:
I considered the two of these. The first one, I don't know why I didn't do it..
And using a Set might bring me trouble down the road since it doesn't really have a get(), so I figured sticking with a List is better.